### ISSUE TITLE
Space trimming in a code span nested in block quote

### ISSUE THREAD

**Issue by mity** [mity/md4c#70]
Currently spaces from code span are not trimmed correctly in this complex case:

```
> ` this 
> is     
> a long 
> code   
> span   
> `      
```

It generates:
``` html
<blockquote>
<p><code>this is a long code span </code></p>
</blockquote>
```

(The final space should not be present.)

It works if not nested in a block-quote:
```
` this   
is       
a long   
code     
span     
`        
```

(Affects both master and new spec-0.29 branch which does it differently due the new trimming rules.)

**Comment by mity** [comment]
For spec-0.29, fixed in b7d3c1a37e459d32591a641128b12f4dc3e00cff.

**Comment by mity** [comment]
Hmm. Fixing in master would require bigger change then I like. Given the 0.29 reworks the code span space trimming rules anyway and that this is sort of corner case, maybe the best strategy is to just live with it until spec-0.29 branch gets merged into master.

**Comment by mity** [comment]
Fixed by merging spec-0.29 branch into master.
